src/components/Animation.scss

```diff
@@ -1,7 +1,7 @@
 /* @getethos/design-system/Animation.scss
    ========================================================================== */
 
-:root {
+.body {
```

Comment:
I'm a little concerned with doing this in the main application and CMS, so we should test it first to make sure there are no regressions. Before we make this an official EDS version I think we need to do the following tests, with the package.json files pointed to this branch:

`"ethos-design-system": "github:getethos/ethos-design-system#arnas/GC-2299"` 

1. Open a CMS PR, find the <body> tag that needs this class, make sure there are no conflicts.
2. Open a main app PR for `/partners/bond/package.json` and `/frontend/package.json`. Again find the <body> tags (may be more than 2) and make sure there are not conflicts.

If you don't have access to main app `github.com/getethos/ethos`, can you submit an IT ticket request for read/write access there? We'll get you onboarded there next week.

As for the nextJS repo, I'm going to block this PR for now but we can use this EDS branch for our dev work.